Maple2Storage/Types/Coord.cs

```diff
@@ -213,16 +213,18 @@ public static CoordB From(sbyte x, sbyte y, sbyte z)
 
         public readonly CoordF ToFloat()
         {
-            return CoordF.From(X, Y, Z);
+            return CoordF.From(
+                (float) (X * Block.BLOCK_SIZE),
+                (float) (Y * Block.BLOCK_SIZE),
+                (float) (Z * Block.BLOCK_SIZE));
```

Comment:
Will this break compatibility with existing usages?

Reply:
It shouldn't because nothing else was using it.